react/features/shared-video/functions.js

```diff
@@ -45,3 +45,22 @@ export function isVideoPlaying(stateful: Object | Function): boolean {
         && (p.name === VIDEO_PLAYER_PARTICIPANT_NAME || p.name === YOUTUBE_PLAYER_PARTICIPANT_NAME))
     );
 }
+
+/**
+ * Returns true if the link is from youtube.
+ *
+ * @param {string} urlToParse - The link to be checked if it is from youtube.
+ * @returns {boolean}
+ */
+export function validateYouTubeUrl(urlToParse: string) {
```

Comment:
@tudor-phd Just realized that there is already a method `getYoutubeId` that validates the URL and is used by mobile. Can you please use that for web as well instead of creating another one ?

Reply:
@tudor-phd at line 125 in SharedVideoDialog.js, i.e.,
`onPostSubmit(youtubeId || link);` shouldn't this be changed to `onPostSubmit(link)` only? Because the variable `youtubeId` outputs the 11-character length YouTube video id while the variable `link` contains the whole URL? It worked in my case when I changed to the latter. Is it possible that I am missing something here? thanks!

Reply:
Hm, I tried your proposal on my local machine and is not working. The main idea of this PR is to prevent sharing other links than youtube links.